libs/input/src/lib/password-field/password-field.stories.tsx

```diff
@@ -25,7 +25,7 @@ const meta: Meta<typeof PasswordField> = {
     },
     color: {
       description: '`ColorVariant`',
-      options: Object.keys(color),
+      options: Object.keys(colorWrapper('text')),
```

Comment:
skip to use `colorWrapper` when we can it will affect performance. You can use `colorCode`
```suggestion
      options: Object.keys(colorCode),
```